src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -58,7 +58,8 @@ interface ReplyProps {
   environment: Environment
   onScroll: () => void
   refetch: RelayRefetchProp["refetch"]
-  openInquiryModal: () => void
+  createsOfferOrder: boolean
```

Comment:
Same case as in ConversationCTA. I don't think we need `createsOfferOrder` here (`openInquiryModal` is perfect, though)